openquake/qa_tests_data/scenario/case_13/job.ini

```diff
@@ -1,7 +1,7 @@
 [general]
 
 description = Multi-rupture scenario
-calculation_mode = scenario
+calculation_mode = event_based
```

Comment:
Why the change in the `calculation_mode` here?

Reply:
Because it does not work for scenario yet, there will be another PR for that (the case of ruptures of different TRTs).